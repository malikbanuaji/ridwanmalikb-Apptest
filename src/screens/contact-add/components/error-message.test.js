import * as React from 'react';
import ErrorMessage from './error-message';

import {render} from '@testing-library/react-native';

test('should render properly', () => {
  const component = render(
    <ErrorMessage errors={{firstName: 'Error'}} name="firstName" />,
  );
  expect(component.toJSON()).toMatchSnapshot();
});

test('should render negative case', () => {
  const component = render(<ErrorMessage errors={{}} name="firstName" />);
  expect(component.toJSON()).toMatchSnapshot();
});
