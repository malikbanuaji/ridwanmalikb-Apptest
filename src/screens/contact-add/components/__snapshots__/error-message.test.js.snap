// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render negative case 1`] = `
<Text
  style={
    Object {
      "color": "#ff3800",
      "fontSize": 14,
      "letterSpacing": 0.1,
    }
  }
/>
`;

exports[`should render properly 1`] = `
<Text
  style={
    Object {
      "color": "#ff3800",
      "fontSize": 14,
      "letterSpacing": 0.1,
    }
  }
/>
`;
